sql-plugin/src/main/330until340/scala/org/apache/spark/sql/rapids/shims/RapidsErrorUtils.scala

```diff
@@ -61,7 +62,7 @@ object RapidsErrorUtils extends RapidsErrorUtilsFor330plus {
     )
   }
 
-  def throwSparkDateTimeException(infOrNan: String): Unit = {
+  def sparkDateTimeException(infOrNan: String): SparkDateTimeException = {
     ShimTrampolineUtil.dateTimeException(infOrNan)
```

Comment:
Is it necessary to leverage `ShimTrampolineUtil` here?  The code here is under org.apache.spark.sql which is a subset of org.apache.spark, so not sure why we need to use org.apache.spark.rapids.himTrampolineUtil here.  Code would be simpler to read and smaller if we could avoid it.